shared/src/test/scala/mouse/OptionSyntaxTest.scala

```diff
@@ -23,6 +23,7 @@ package mouse
 
 import scala.util.{Failure, Success}
 
+@annotation.nowarn("cat=deprecation")
```

Comment:
Can we annotate a particular test suite instead of the whole test class?

Reply:
`test` is just a method call so we could not use an annotation on them, maybe there is another way but I do not know of it

Reply:
Something like this should work fine
```scala
@nowarn("cat=deprecation")
val resultRight = Option(3).right("S")
assertEquals(resultRight, Option(3).toRight("S"))
```
And accordingly this too (but looks not so cool)
```scala
@annotation.nowarn("cat=deprecation")
val rightTest = test("OptionSyntax.right") { ... }
```